packages/api-server/src/app/cluster.ts

```diff
@@ -5,11 +5,14 @@ import { logger } from "../base/logger";
 import { BlockEmitter } from "../block-emitter";
 import { CKBPriceOracle } from "../price-oracle";
 import { initSentry } from "../sentry";
+import { startOpentelemetry } from "../opentelemetry";
 
 const numCPUs = cpus().length;
 const clusterCount = +(envConfig.clusterCount || 0);
 const numOfCluster = clusterCount || numCPUs;
 
+startOpentelemetry();
```

Comment:
do we need `startOpentelemetry()` in every process? is there any problem if we move this into `cluser.isMaster` branch?

Reply:
Yes. If so, only master process export trace.